src/dotnet/ToolManifest/ToolManifestFile.cs

```diff
@@ -52,7 +51,8 @@ public IReadOnlyCollection<ToolManifestPackage> Find(FilePath? filePath = null)
                     DeserializeLocalToolsManifest(possibleManifest);
 
                 List<ToolManifestPackage> toolManifestPackageFromOneManifestFile =
-                    GetToolManifestPackageFromOneManifestFile(deserializedManifest, possibleManifest, correspondingDirectory);
+                    GetToolManifestPackageFromOneManifestFile(deserializedManifest, possibleManifest,
```

Comment:
I always prefer to break each parameter on a newline when one goes to a new line. Is this something editorconfig could to for us? I have been meaning to add that to the CLI repos.

Reply:
That would be nice. I am not aware of editorconfig option. I just run auto format and get this. The editor extension i use will just make a new line after command once it go over 80 char.

Reply:
I think it might be hard to do automatically 

`Add(1,2,3,4,5....81);` will be formatted to the following by my editor current rule
```
Add(1,2,3,4,5....80,
           81); 
```

but if we enforce one line per param, it will look like
```
Add(
  1,
  2,
  3,
  .
  .
  .
  81
)
```

Reply:
What's wrong that result?

(IMHO, 80 chars is way too limiting for C#. It's a holdover from when printing code to paper was common, screens and terminals were limited, and languages like C where you don't lose two levels of indentation to namespace and class and tend to abbreviate everything. If we're going to enforce a limit, it should be something like 100 or even 120. I think the point at which things scroll in GitHub is around there.)

Reply:
There is no editorconfig setting for wrapping parameter lists. There is a community PR (https://github.com/dotnet/roslyn/pull/30422) to add a refactoring that will wrap lists of things, but that would still be a manual task.

Reply:
I personally use 120, so, that's my vote.

Reply:
fixed